tvb_epilepsy/base/model/disease_hypothesis.py

```diff
@@ -39,9 +38,11 @@ def __init__(self, number_of_regions, excitability_hypothesis={}, epileptogenici
             self.name = self.type + "_Hypothesis"
         else:
             self.name = name
-
-        self.propagation_indices = np.array(propagation_indices)
-        self.propagation_strengths = np.array(propagation_strenghts)
+        self.lsa_propagation_indices = np.array(lsa_propagation_indices)
+        self.lsa_propagation_strengths = np.array(lsa_propagation_strenghts)
+        self.context_str = "from " + construct_import_path(__file__) + " import DiseaseHypothesis, shorten_values"
+        self.create_str = "DiseaseHypothesis(" + str(self.number_of_regions) + ", name='" + self.name+ "')"
+        self.transform_str = "shorten_values(obj)"
```

Comment:
The last 3 attributes from __init__ are written in H5 files and used to create objects at read time. This is not a nice solution and should be changed.
Also, this solution is used for all the model classes and service classes for which we write H5 files. This should be corrected everywhere.